### DISCUSSION TITLE
Hebrew translations

### DISCUSSION THREAD

**Discussion by Ori-Roza** [fastapi/fastapi#9809] [Translations]
Hi everyone,

We are starting this thread for tracking and coordinating hebrew translation.

Contribution
Please let us know what page you are interested in translating so that we can coordinate
Put up a PR, tag this issue in the PR and wait for reviews
fix potential issues and get it merged
When you face potential issues during translation of technical terms, like docker, container etc feel free to put it here.

**Comment by tiangolo** [comment]
Hello! I had created another discussion for this a while back here: https://github.com/tiangolo/fastapi/discussions/9199, so I'm gonna close this one to keep only one. Thanks! 🚀
